products/shopware.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+versionCommand: php bin/console --version
+# releasePolicyLink: https://shopware.com/
+changelogTemplate: "https://github.com/shopware/shopware/blob/__LATEST__/CHANGELOG.md"
+releaseDateColumn: true
+activeSupportColumn: Bug Fix Maintenance
+eolColumn: Security Maintenance
+extendedSupportColumn: Extended Support
+
+auto:
+-   git: https://github.com/shopware/platform.git
```

Comment:
They renamed the repo to https://github.com/shopware/shopware.git some time ago; maybe good to use the new location? (see also https://www.shopware.com/en/news/shopware-github-repository-will-move/)

Same goes for any other place where https://github.com/shopware/platform is referenced.